advanced_functionality/tensorflow_bring_your_own/container/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# https://docs.docker.com/compose/gettingstarted/#step-2-create-a-dockerfile
+FROM tensorflow/tensorflow:1.8.0-py3
+
+RUN apt-get update && apt-get install -y --no-install-recommends nginx curl
+
+# Download tensorflow serving
```

Comment:
You can do it after merging: add link from tf serving docs https://www.tensorflow.org/serving/setup#installing_the_modelserver